libs/counterfactuals/src/util/getCurrentLabel.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function getCurrentLabel(
+  taskType: "classification" | "regression" | "text_classification",
+  desiredRange?: [number, number],
+  desiredClass?: string
+): string {
+  if (taskType === "regression") {
```

Comment:
same here, move task type "regression" to some global constants

Reply:
Updated to DatasetTaskType.Regression